packages/explorer-2.0/components/Orchestrators/PerformanceTable.tsx

```diff
@@ -0,0 +1,539 @@
+import { Flex, Box } from 'theme-ui'
+import { useMemo } from 'react'
+import { useTable, useFilters, useSortBy, usePagination } from 'react-table'
+import Search from '../../public/img/search.svg'
+import Help from '../../public/img/help.svg'
+import matchSorter from 'match-sorter'
+import AccountCell from '../AccountCell'
+import ReactTooltip from 'react-tooltip'
+import useWindowSize from 'react-use/lib/useWindowSize'
+import Router from 'next/router'
+import { MdKeyboardArrowUp, MdKeyboardArrowDown } from 'react-icons/md'
+import { RiArrowLeftLine, RiArrowRightLine } from 'react-icons/ri'
+
+const PerformanceTable = ({ data: { currentRound, transcoders }, region }) => {
+  const { width } = useWindowSize()
+  function fuzzyTextFilterFn(rows, id, filterValue) {
+    return matchSorter(rows, filterValue, {
+      keys: [(row) => row.values[id]],
+    })
+  }
+
+  // Let the table remove the filter if the string is empty
+  fuzzyTextFilterFn.autoRemove = (val) => !val
+
+  function DefaultColumnFilter({ column: { filterValue, setFilter } }) {
+    return (
+      <Flex
+        sx={{
+          alignItems: 'center',
+          pl: 3,
+        }}
+      >
+        <Search sx={{ width: 16, height: 16, mr: 1, color: 'muted' }} />
+        <Box
+          value={filterValue || ''}
+          onChange={(e) => {
+            setFilter(e.target.value || undefined)
+          }}
+          placeholder={`Filter`}
+          as="input"
+          type="text"
+          variant="input"
+          sx={{
+            display: 'block',
+            outline: 'none',
+            width: '100%',
+            appearance: 'none',
+            fontSize: 2,
+            lineHeight: 'inherit',
+            border: 0,
+            color: 'inherit',
+            bg: 'transparent',
+          }}
+        />
+      </Flex>
+    )
+  }
+
+  const columns: any = useMemo(
+    () => [
+      {
+        Header: '#',
+        accessor: 'rank',
+        disableSortBy: true,
+      },
+      {
+        Header: 'Orchestrator',
+        accessor: 'id',
+        filter: 'fuzzyText',
+        Filter: DefaultColumnFilter,
+        mobile: true,
+        sortType: (rowA, rowB, columnID) => {
+          let a = getRowValueByColumnID(rowA, columnID)
+          let b = getRowValueByColumnID(rowB, columnID)
+          let aThreeBoxSpace = getRowValueByColumnID(rowA, 'threeBoxSpace')
+          let bThreeBoxSpace = getRowValueByColumnID(rowB, 'threeBoxSpace')
+          let rowAIdentity = aThreeBoxSpace?.name ? aThreeBoxSpace?.name : a
+          let rowBIdentity = bThreeBoxSpace?.name ? bThreeBoxSpace?.name : b
```

Comment:
I think we can combine L73, L75 and L77 into one line (and do the same for row B)

```suggestion
          let rowAIdentity =  getRowValueByColumnID(rowA, 'threeBoxSpace')?.name || getRowValueByColumnID(rowA, columnID)
          let rowBIdentity = getRowValueByColumnID(rowB, 'threeBoxSpace')?.name || getRowValueByColumnID(rowB, columnID)
```

Reply:
done